enterprise/api/v1/views.py

```diff
@@ -771,7 +771,7 @@ def cancel_enrollment(self, request, fulfillment_source_uuid):
         except Exception as exc:  # pylint: disable=broad-except
             msg = (
                 f'Subsidized enrollment terminations error: unable to unenroll User {username}'
-                f'from Course {enrollment.course_id}  because: {str(exc)}'
+                f'from Course {enrollment.enterprise_course_enrollment.course_id}  because: {str(exc)}'
```

Comment:
This change wasn't actually needed/related to this PR, but I did run into it.  The enrollment here has no course_id attribute.

Reply:
nit: `enrollment` seems like not the right variable name in here - isn't it really a fulfillment?

Reply:
totally, i can change that too